streams/voxel_stream_region_files.cpp

```diff
@@ -880,7 +880,7 @@ void VoxelStreamRegionFiles::_convert_files(Meta new_meta) {
 		int lod;
 	};
 
-	ERR_FAIL_COND(old_stream->load_meta() != OK);
+	ERR_FAIL_COND(old_stream->load_meta() != (int)OK);
```

Comment:
Something's off here. Cast should not have been needed.

Reply:
VOXEL_FILE_OK fixed it.